oci_python_image/integration_test/__test__.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+import pytest
+
+if __name__ == "__main__":
```

Comment:
note, this is what the aspect_rules_py `py_pytest_main` creates. That repo isn't on registry.bazel.build yet, so I just vendored the file content here.